### DISCUSSION TITLE
PK-sim population

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1396] [Questions & Problems]
Hello everyone, I have been using PK-sim for modeling recently. The last application part is to simulate a virtual crowd. I would like to ask if there is a built-in virtual adult crowd in PK-sim. Looking forward to your reply!

**Comment by msevestre** [comment]
Do you mean a population? 
If yes, please read the online manual on creating a population. A population is a standard PKsim building block. If not , please clarify your question

**Comment by WTY-tt** [comment]
Hello, What I just said may not be very clear.I am familiar with the group building module. I want to ask if there is an adult group that comes with PK-sim that can be used directly.

**Reply by WTY-tt** [reply]
Hello, What I just said may not be very clear.I am familiar with the group building module. I want to ask if there is an adult group that comes with PK-sim that can be used directly.
